docs/spatial-vector.rst

```diff
@@ -1,4 +1,34 @@
 Earthpy Spatial Vector Data
 ===========================
 
-Edits go here. 
+The ``earthpy`` spatial module provides functions that wrap around the ``rasterio``
+and ``geopandas`` to work with raster and vector data in Python.
+
+Clip Vector Data
+~~~~~~~~~~~~~~~~
+
+The ``clip_shp`` function takes a geopandas dataframe and a point or bounding box clip object and spatially clips the dataframe to the clip object.
```

Comment:
@lwasser I've made these changes and pushed them to my repo. I  rewrote it slightly because I was confused by "The first object is a point, line of polygon GeoDataFrame". Let me know if my slight rewrite is okay.